### PR TITLE
Add 7-piece Syzygy tablebases

### PR DESCRIPTION
**Pull request by niklasf**
## What kind of PR is this?
**Select only one** *to speed up review/approval; i.e., don't lump an addition and a removal into a single PR.*

- [x] **Adding** something new to awesome-ipfs
- [ ] **Editing** something already listed on awesome-ipfs
- [ ] **Removing** something from awesome-ipfs
- [ ] **Something else** *(if so, please explain in the "Additional details" section below)*

## Pre-submit checklist
**Please confirm ALL of the following** before submitting your PR.

- [x] This PR includes only one addition, removal, or edit.
- [x] I edited the `/data` directory instead of the [README.md](https://github.com/ipfs/awesome-ipfs/blob/master/README.md).
- [x] I ran the `make build` command following my edits to the `/data` directory.
- [x] I reviewed the [content policy](https://github.com/ipfs/awesome-ipfs/blob/master/POLICY.md) and the and the [IPFS Community Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md) to ensure my submission meets the requirements.
- [x] I have followed the [CONTRIBUTING.md guidelines](https://github.com/ipfs/awesome-ipfs/blob/master/CONTRIBUTING.md).

### Is there anything else we should know about this PR?
It might be intresting that this appears to be the largest data set yet, in this repository.

### For new additions: What is your project's *current* size/reach?
Syzygy tablebases are used to analyse games in the chess community. The tablebase server hosted by lichess.org is hit with 5-10 probes per second, depending on the time of day. Most of these are triggered by a user interaction (like making a move on an analysis board), but the number of users is not tracked.

### For new additions: What is your project's *potential* size/reach?
Due to the difficulty of downloading very large tablebase files from traditional mirrors, the majority of users do not have offline access to Syzygy tables. IPFS enables dowloading only the relevant blocks for offline analysis, potentially allowing all chess players to analyse games with tablebases, even where a stable internet connection is not available.

### For new additions: How critical is IPFS to your project?

- [ ] Essential
- [ ] Somewhat critical
- [x] Useful, but not critical

### For new additions: What core goal(s) does your project address?

- [ ] Big-data solutions
- [ ] Freedom from corporate/government interference
- [x] Data integrity
- [x] Dev tools or other ways to enable developers
- [ ] Disaster resilience/recovery
- [x] File storage/retrieval
- [x] File streaming
- [x] Permanent archiving
- [ ] Self-sovereign idenity
- [ ] Platforms that utilize consensus, reputation, or incentivization
- [ ] Other *(please list)*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 > Useful resources for using [IPFS](https://ipfs.io) and building things on top of it
 
-_This list is for projects, tools, or pretty much any things related to IPFS that are totally_ **awesome**_. This is for products which are already awesome - if you have plans for cool stuff to do with IPFS, you should build it, and then link it here. If you have an idea for an awesome thing to do with IPFS, a good place to ask about it might be in [our IPFS Discuss forums](https://discuss.ipfs.io).
+_This list is for projects, tools, or pretty much any things related to IPFS that are totally_ **awesome**_. This is for products which are already awesome - if you have plans for cool stuff to do with IPFS, you should build it, and then link it here. If you have an idea for an awesome thing to do with IPFS, a good place to ask about it might be in [ipfs/apps](https://github.com/ipfs/apps) or [ipfs/notes](https://github.com/ipfs/notes)._
```
Comment (niklasf) [inline README.md]: This unrelated change was made automatically by the build script.

Comment (hsanjuan) [reply README.md]: sorry, this was my fault. Any chance that you can rebase on master and rebuild?

Comment (niklasf) [reply README.md]: Done :)
